retrieve_MC3_mutations.py

```diff
@@ -36,27 +40,28 @@
 
 # simple mutation dictionary {cancer_type: {tcga_id: ["PTEN", "TP53"]}}
 # this will be used at end to create simple 0/1 mutation status data frame
-mutation_dict = {x:{} for x in cancer_types_of_interest}
+mutation_dict = {x: {} for x in cancer_types_of_interest}
 
 # open up MC3 and define header lines
 mc3 = gzip.open(mc3_filename, "rb")
```

Comment:
Do you need to close this file handle? Not sure how the Python `gzip` package works, maybe it automatically closes them.